.travis.yml

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 ChainSafe Systems
+# SPDX-License-Identifier: LGPL-3.0-only
+
+env:
+  global:
+    - GO111MODULE=on
```

Comment:
why?

Reply:
I don't think installing geth affects if go mods is enabled or not (from make install_deps)

Reply:
This is required to enable go mods